pkg/cmd/server/start.go

```diff
@@ -180,6 +183,26 @@ func (o WardleServerOptions) RunWardleServer(stopCh <-chan struct{}) error {
 		return err
 	}
 
+	// add http-cache
+	memcached, err := memory.NewAdapter(
+		memory.AdapterWithAlgorithm(memory.LRU),
+		memory.AdapterWithStorageCapacity(500000000), // 500MB
+	)
+	if err != nil {
+		logger.L().Fatal("failed to create memcached adapter", helpers.Error(err))
+	}
+
+	cacheClient, err := cache.NewClient(
+		cache.ClientWithAdapter(memcached),
+		cache.ClientWithTTL(10*time.Minute),
```

Comment:
obviously this needs to be set lower, or we should check how to invalidate cache

Reply:
Yes indeed